examples/deletecard/app.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import Announcer from '../../src/Announcer';
+
+class App extends Component {
+  constructor(props) {
```

Comment:
It might be worth it just as a fun exercise to make these use hooks instead of being class components.